.github/workflows/1_2_b_bump_extension_only.yml

```diff
@@ -47,7 +47,7 @@ jobs:
 
       - name: Checkout previous branch
         if: endsWith(github.ref, '/main') == false
-        run: git checkout ${{ github.ref_name }}
+        run: git checkout -f ${{ github.ref_name }}
```

Comment:
ℹ️ Using `-f` to also checkout if that would overwrite any changes files. E.g. slight differences in `yarn.lock` or the like.